### DISCUSSION TITLE
How is efficiency calculated

### DISCUSSION THREAD

**Discussion by monojk** [tillsteinbach/VWsFriend#103] [Q&A] [answered]
The Grafana Car Overview shows an "efficiency" plot. How ist that calculated or what does it mean? I only see an efficiency of about 70% *- 10%

**Comment by tillsteinbach** [comment, marked as answer]
It is an artificial value. During configuration you set the WLTP value of the car. This is 100%. So if you get 70% this means: The consumption is in a way you would reach 70% of the WLTP range of your car if it would be charged 100%.
